inspirehep/modules/records/serializers/__init__.py

```diff
@@ -82,37 +76,10 @@
 )
 
 bibtex_v1 = PybtexSerializerBase(PybtexSchema(), BibtexWriter())
-latexeu_v1 = LATEXEUSerializer()
-latexus_v1 = LATEXUSSerializer()
-cvformatlatex_v1 = CVFORMATLATEXSerializer()
-cvformathtml_v1 = CVFORMATHTMLSerializer()
-cvformattext_v1 = CVFORMATTEXTSerializer()
```

Comment:
this also renders most of the code in `inspirehep.modules.records.serializers` unused, so they could be removed (maybe in a different PR?)

Reply:
All code responsible for those serializers is also removed from `inspirehep.modules.records.serializers` did I missed something?

Reply:
sorry, the code doing the actual work seems to be in `inspirehep.modules.utils` (look at the imports of the files you removed).